tests/ctst/features/iam-policies/AssumeRoleWithWebIdentity.feature

```diff
@@ -0,0 +1,144 @@
+Feature: Assume Role with Web Identity
+    In order to interact with restricted APIs
+    As an Artesca User
+    I want to use a web identity
+    And to subscribe to events I want to be notified on
+    And to receive notifications on buckets/objects activities I have subscribed to
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-ARWWI
+  Scenario Outline: Assume Role with Web Identity
+    Given an existing bucket "<bucketName>" "<withVersioning>" versioning
+    And we "<haveObject>" an existing object "<objectName>" "<withVersionId>" version-Id "<objectVersionId>"
+    And a <type> type
+    When the user tries to perform "<ifS3Standard>" "<action>" on the bucket
+    Then the user should be able to perform successfully
```

Comment:
Not asking for change here, but just an idea to reduce the number of lines here:
Related to the above comment, if we want to test the APIs with all identities, we can use a data table for all types we want, and then, in the code, we can call sequentially the right prepareForType() and then the API call in a loop, and store everything in `this.results`, the benefit is that we can, this way, automatically test any API as all identity types